slash/react/src/List/ContentItemDuo/ContentItemDuo.tsx

```diff
@@ -0,0 +1,72 @@
+import checkIcon from "@material-symbols/svg-400/outlined/check.svg";
+import closeIcon from "@material-symbols/svg-400/outlined/close.svg";
+import { useMemo } from "react";
+import { Variants } from "../../Button/Button.client";
+import { getComponentClassName } from "../../Form/core";
+import { Button, Svg } from "../../client";
+
+type ContentItemDuoProps = {
+  label: string;
+  value: string;
+  isVertical?: boolean;
+  className?: string;
+  classModifier?: string;
+  isShowingDoneIcon?: boolean;
+  isShowingCloseIcon?: boolean;
+  buttonText?: string;
+  onButtonClick?: () => void;
+};
+
+export const ContentItemDuo = ({
+  label,
+  value,
+  isVertical = false,
+  className,
+  classModifier,
+  isShowingDoneIcon = false,
+  isShowingCloseIcon = false,
+  buttonText,
+  onButtonClick,
+}: ContentItemDuoProps) => {
+  const componentClassName = useMemo(() => {
+    let newClassModifier = classModifier ?? "";
+
+    if (isVertical) {
+      newClassModifier += " vertical";
+    }
+
+    return getComponentClassName(
+      className,
+      newClassModifier,
+      "af-content-item-duo",
+    );
```

Comment:
```typescript
const classMoifiers = [classModifier];

if (isVertical) {
  classModifiers.push("vertical");
}

return getComponentClassName(
  className,
  classModifiers.filter(Boolean).join(" "),
  "af-content-item-duo",
);

Reply:
Resolved in https://github.com/AxaFrance/design-system/pull/450/commits/1768e32f9f7f1c7dbe11dbb9d3a442d88a067ccb